### PR TITLE
feat: Micro resource

### PR DESCRIPTION
**Pull request by yoshwata**
## Context
When a lot of pods of LOW resources are launched for build at once, the cluster resource drys up. That happens at our cluster actually, so it is PR to avoid such depletion.

## Objective
Add MICRO pod resource.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -160,9 +164,23 @@ class K8sExecutor extends Executor {
...
-        const MEMORY = (annotations[RAM_RESOURCE] === 'HIGH') ? this.highMemory : this.lowMemory;      // 12GB or 2GB
+        const CPU = (cpuConfig in cpuValues) ? cpuValues[cpuConfig] * 1000 : cpuValues.MICRO * 1000;
+
+        const memValues = {
+            HIGH: this.highMemory,
+            LOW: this.lowMemory,
+            MICRO: this.microMemory
+        };
+        const memConfig = annotations[RAM_RESOURCE];
+        const MEMORY = (memConfig in memValues) ? memValues[memConfig] : memValues.MICRO;
```
Comment (tk3fftk) [inline index.js]: Should we set the default setting to `MICRO`?

Comment (yoshwata) [reply index.js]: I talked with @catto. This should be LOW not to break the original behavior. Thanks.

File: test/index.test.js [hunk 2]
```diff
@@ -337,8 +343,9 @@ describe('index', function () {
         });
```
Comment (wdstar) [inline test/index.test.js]: Missing test case for `MICRO` settings.

File: test/index.test.js [hunk 3]
```diff
@@ -339,6 +345,7 @@ describe('index', function () {
         it('sets the memory appropriately when ram is set to HIGH', () => {
             postConfig.json.metadata.cpu = 2000;
             postConfig.json.metadata.memory = 12;
+            fakeStartConfig.annotations['beta.screwdriver.cd/cpu'] = 'LOW';
```
Comment (wdstar) [inline test/index.test.js]: `LOW` setting is extra, because it is default setting. The same applies to the following test cases.

Comment (yoshwata) [reply test/index.test.js]: There is no need to set default `LOW`. I remove those. Thank you.

**Pull request merged**